src/load_obfuscation_shims.c

```diff
@@ -6,6 +6,13 @@
 extern s8 obfuscated_obfuscation_shims_ROM_START[];
 extern s8 obfuscated_obfuscation_shims_VRAM[];
 
+#ifdef VERSION_JP
+extern Addr battle_heap_create_obfuscated_ROM_START;
```

Comment:
these should be generated and placed in ld_addrs if the segment is named properly in the yaml. I think that'd be better than explicitly putting them here